src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/Indexing.scala

```diff
@@ -46,95 +45,64 @@ private class Indexing(resources: Resources[Task], cache: DistributedCache[Task]
 
   def startKafkaStream(): Unit = {
 
-    def processProject(accountRef: AccountRef, projectRef: ProjectRef): Boolean => Task[Unit] = {
-      case true =>
-        coordinator ! Msg(accountRef, projectRef)
-        Task.unit
-      case false =>
-        Task.raiseError(new RetriableErr(s"Failed to update project '${projectRef.id}'"))
-    }
-
-    def processAccount(accountRef: AccountRef): Boolean => Task[Unit] = {
-      case true =>
-        Task.unit
-      case false =>
-        Task.raiseError(new RetriableErr(s"Failed to update account '${accountRef.id}'"))
-    }
-
     def index(event: KafkaEvent): Future[Unit] = {
       val update = event match {
         case OrganizationCreated(_, label, uuid, rev, _, org) =>
           cache
-            .addAccount(AccountRef(uuid), Account(org.name, rev, label, deprecated = false, uuid), updateRev = false)
-            .flatMap(processAccount(AccountRef(uuid)))
+            .addAccount(AccountRef(uuid), Account(org.name, rev, label, deprecated = false, uuid))
 
         case OrganizationUpdated(_, label, uuid, rev, _, org) =>
           cache
-            .addAccount(AccountRef(uuid), Account(org.name, rev, label, deprecated = false, uuid), updateRev = true)
-            .flatMap(processAccount(AccountRef(uuid)))
+            .addAccount(AccountRef(uuid), Account(org.name, rev, label, deprecated = false, uuid))
         case OrganizationDeprecated(_, uuid, rev, _) =>
-          cache.deprecateAccount(AccountRef(uuid), rev).flatMap(processAccount(AccountRef(uuid)))
+          cache.deprecateAccount(AccountRef(uuid), rev)
         case ProjectCreated(_, label, uuid, orgUUid, rev, _, proj) =>
-          cache
-            .addProject(
+          for {
```

Comment:
is this correct? Isn't supposed to not continue attempting to cache a view if the cache project return false?
Same for 
```scala
coordinator ! Msg(AccountRef(orgUUid), ProjectRef(uuid))
```
This should be called only when the project cache returns true, the view cache returns true and the resolver cache returns true